admin/tabs/general/alertBanner/AlertBannerModel.ts

```diff
@@ -104,6 +108,85 @@ export class AlertBannerModel extends HoistModel {
         this.refreshAsync();
     }
 
+    loadPreset(preset: PlainObject) {
+        this.formModel.setValues({...preset, expires: null});
+    }
+
+    @action
+    addPreset() {
+        const {message, intent, iconName, enableClose} = this.formModel.values,
+            dateCreated = Date.now(),
+            createdBy = XH.getUsername();
+        this.savedPresets = sortBy(
+            [
+                ...this.savedPresets,
+                {message, intent, iconName, enableClose, dateCreated, createdBy}
+            ],
+            ['intent', 'message']
+        );
+        this.savePresetsAsync();
+    }
+
+    @action
+    async removePreset(preset: PlainObject) {
+        const confirmRemove = await XH.confirm({
+            message: fragment(
+                p(
+                    'You are removing a preset banner configuration. A user may have saved this, see the preset subtext for the creator.'
+                ),
+                p(
+                    'Choose below whether you would like to remove this configuration from the admin list of saved presets.'
+                )
+            ),
```

Comment:
Looks good on the overall use of the confirm here.  I'd simplify the message a bit - we know that some user definitely _did_ save the preset, so not getting too much from the longer phrasing.  Could go with "Are you sure you wish to this delete this preset?"

Reply:
also -- along the lines of keeping things simple -- don't think we need to customize the buttons.  This is not *that* dangerous.  Idea is, you need a good reason to go beyond the hoist defaults, which hopefully are good for most cases....